web/src/pages/billing/BillingInvoiceMonthCost.tsx

```diff
@@ -477,7 +702,7 @@ const BillingCurrentCost = () => {
                     <Grid>
                         <Grid.Column width={12}>
                             <HorizontalStackedBarChart
-                                data={costRecords}
+                                data={chartData}
```

Comment:
I can't comment on the actual line cos it isn't changed but it'd be good to have a loader when in table view when things are loading. Replacing:

```tsx
{!showAsChart ? (
    <Table celled compact sortable>
```

with 

```tsx
{!showAsChart && isLoading ? <LoadingDucks /> : null}

{!showAsChart && !isLoading ? (
    <Table celled compact sortable>

```

seems to work